lib/factories/ppt/index.js

```diff
@@ -50,13 +50,34 @@ class PptFactory {
         return this.slideFactory.addSlide(slideName, layoutName);
     }
 
-    addImage(slide, filePath, options) {
+    addImage(slide, filePath, options = {}) {
         let mediaName = `image-${uuidv4()}${path.extname(filePath)}`;
         let rId = this.slideFactory.addImageToSlideRelationship(slide, `../media/${mediaName}`);
 
+        if (options.url) {
```

Comment:
Is this general purpose enough to be extracted into a helper lib that other elements that have hyperlinks can use?

Reply:
Was contemplating this too.

Reply:
Note: the setting up of the rels for a link can be done in a helper, but not the actual XML node generation. There's too much of a variation between where link nodes get inserted and how they look.